aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -36,7 +36,7 @@ Each circuit uses approximately 250 KB of memory for a minimal *Hello World*-sty
 
 ## SignalR configuration
 
-Blazor Server apps use [ASP.NET Core SignalR](<xref:signalr/introduction>) to communicate with the browser. [SignalR's hosting and scaling conditions](xref:signalr/publish-to-azure-web-app) apply to Blazor Server apps.
+Blazor Server apps use [ASP.NET Core SignalR](xref:signalr/introduction) to communicate with the browser. [SignalR's hosting and scaling conditions](xref:signalr/publish-to-azure-web-app) apply to Blazor Server apps.
```

Comment:
@guardrex You might've seen that I've been doing a daily run of fixing up our `/dotnet/api` references across the repo. I've finally finished, so I also added a commit to remove the `<` and `>` around an xref that's got custom text. This is the only place I've touched anything under the Blazor node.